### PR TITLE
fix #76230 mopo.de

### LINKED ISSUE #76230: mopo.de
### Issue URL (Ads)
 [https://www.mopo.de/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.mopo.de%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.1.0
Browser: | Safari
Simplified filters: | disabled
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Language-specific:</b><br/> AdGuard German, <br/>EasyList Germany

### PR DESCRIPTION
**Pull request by stanislav-atr**
#76230

### CODE REVIEW

File: GermanFilter/sections/specific.txt [hunk 1]
```diff
@@ -104,6 +104,8 @@ focus.de##.articleContent > .textBlock > h2:empty
 wrestling-infos.de##center > a[target="_blank"]:not([rel]) > img[alt^="http"]
 golem.de##.list-articles > li[class*="li-hb-"]
 hausbau-forum.de##.samItem
+||mopo.de/assets/*/Gd/desktop/js/dmn-main.js
```
Comment (Sergey-Lyapin) [inline GermanFilter/sections/specific.txt:107]: I'm not sure we should block this request

File: GermanFilter/sections/specific.txt [hunk 2]
```diff
@@ -104,6 +104,8 @@ focus.de##.articleContent > .textBlock > h2:empty
...
+mopo.de##body > div[class] > img[referrerpolicy="unsafe-url"]
```
Comment (Sergey-Lyapin) [inline GermanFilter/sections/specific.txt:108]: please change to `img[referrerpolicy="unsafe-url"]` and fix "Ad" caption under the ad

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

**Comment by BlazDT** [comment]
Those are adblock detection ads in screenshot.

File: GermanFilter/sections/specific.txt (hunk 2 above)
Comment (stanislav-atr) [reply GermanFilter/sections/specific.txt:108]: For some reason 
`mopo.de##img[referrerpolicy="unsafe-url"]
`and
`mopo.de##div[class] > img[referrerpolicy="unsafe-url"]
`dont remove that banner on my device.

Comment (Alex-302) [reply GermanFilter/sections/specific.txt:108]: Better check `mopo.de#@#.AD-POST`

Comment (Alex-302) [reply GermanFilter/sections/specific.txt:108]: added

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**